vault/logical_system_paths.go

```diff
@@ -1768,7 +2183,17 @@ func (b *SystemBackend) policyPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ListOperation: &framework.PathOperation{
 					Callback: b.handlePoliciesPasswordList,
-					Summary:  "List the existing password policies.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"keys": {
+									Type: framework.TypeStringSlice,
```

Comment:
```suggestion
									Type: framework.TypeStringSlice,
									Required: false
```
* the suggestion may cause go fmt errors